Crypto/testsuite/src/CryptoTestSuite.cpp

```diff
@@ -10,7 +10,7 @@
 
 #include "Poco/Platform.h"
 // see https://github.com/openssl/openssl/blob/master/doc/man3/OPENSSL_Applink.pod
-#if defined(POCO_OS_FAMILY_WINDOWS)
+#if defined(_MSC_VER)
```

Comment:
Sometimes I see, we use `POCO_OS_FAMILY_WINDOWS` and sometimes we use `_MSC_VER` .... May I ask what is the difference?

Reply:
OpenSSL for MinGW has no `#include "openssl/applink.c"`.

Reply:
ok